README.md

```diff
@@ -207,9 +208,22 @@ Submodules needs to be fetched separately via go get.
 
 The memory based event store is part of the main module and does not need to be fetched separately.
 
+### Custom made event store
+
+If you want to store your events in another database beside the already implemented event stores: `sql`, `bbolt`, `esdb` or `memory`, you can implement a custom made event store. It has to implement the following  interface to support the eventsourcing.Repository.
+
+```go
+type EventStore interface {
+    Save(events []core.Event) error
+    Get(id string, aggregateType string, afterVersion core.Version) (core.Iterator, error)
+}
+```
+
+The event store needs to import the `github.com/hallgren/eventsourcing/core` module that expose the `core.Event`, `core.Version` and `core.Iterator` types.
+
 ## Serializer
 
-To store events they have to be serialised into `[]byte`. Default the `encoding/json` is used but it could be replaced on the repoository by setting the `repo.Serializer` and `repo.Deserializer` function properties.
+To store events they have to be serialised into `[]byte`. Default the `encoding/json` is used but it could be replaced on the repository by setting the `repo.Serializer` and `repo.Deserializer` function properties.
```

Comment:
```suggestion
To store events they have to be serialised into `[]byte`. By default the `encoding/json` is used but it could be replaced on the repository by setting the `repo.Serializer` and `repo.Deserializer` function properties.
```